gtk4/sample/examples/application5/Rakefile

```
@@ -0,0 +1,11 @@
+require 'rake/clean'
```

Could you use double quote for consistency?

```suggestion
require "rake/clean"
```